build_tools/github_action/configure_ci.py

```diff
@@ -201,97 +200,81 @@ def get_pr_labels(args) -> List[str]:
 
 
 def matrix_generator(
-    is_pull_request, is_workflow_dispatch, is_push, base_args, families, is_test
+    is_pull_request, is_workflow_dispatch, is_push, base_args, families, is_linux=True
```

Comment:
nit: I'd prefer `platform="linux"` to allow for other platforms eventually. I'm guilty of quite a few of those myself, having just "Windows" and "Not Windows" meaning Linux:

https://github.com/ROCm/TheRock/blob/65a1e46e9847e0ab3b26e071dd952763a0ce5c69/CMakeLists.txt#L136-L147

Bit easier to do platform checks with strings in Python than it is in CMake though.